packages/manager/apps/hpc-vmware-managed-vcd/src/utils/getPricedOrderableResource.ts

```diff
@@ -30,7 +27,7 @@ export const getPricedVdcResources = ({
         ({ planCode }) => planCode === resource.profile,
       )?.prices;
       const pricing = prices?.find(({ capacities }) =>
-        capacities.includes(ProductPricingCapacity.RENEW),
+        capacities.includes('renew'),
```

Comment:
Why not keep a constant here ?

Reply:
It's because types have been updated here : [catalog types](https://github.com/ovh/manager/pull/13751/files/1116081357268ea2fde96aa2f371de9078af5b82#diff-fe246b254e7e8c2a5580055faf61daea730b67bb8ce4a9e15c80e4c36d92404b)
I made this change to avoid having to use the enum & get autocompletion, but I can revert if it doesn't feel right